pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[project]
+name = "keep"
+version = "2.10.1"
+description = "Personal shell command keeper"
+readme = "README.md"
+authors = [
+    { name = "Himanshu Mishra", email = "himanshu.mishra.kgp@gmail.com" },
+]
+dependencies = ["PyGithub", "click", "requests", "terminaltables3"]
```

Comment:
We should pin the version for `PyGithub` to `1.4.7` as was the case in `requirements.txt`

Reply:
it's unpinned in `setup.py` though (which is what users see/install and I agree is the way to go): https://github.com/OrkoHunter/keep/blob/e6eadb926464af1ed41e9c8c69b72882b6bb1d77/setup.py#L28-L33